moderation/preset-reasons.md

```diff
@@ -13,14 +13,14 @@ Save your moderators' time by storing preset reasons to use when making redundan
 ---
 
 ## About
-Preset reasons allow for moderators to save time when issuing punishments by using preset flags for reasons that are repeatedly used (Like canned responses for moderators!)
+Preset reasons allow moderators to save time when issuing punishments by using preset flags for reasons that are repeatedly used (like canned responses for moderators!)
 
 ## Configuration
-Configuring preset reasons for your server is simple. Please keep in mind that you must have the Administrator permission or be a Moderation Manager to configure preset reasons. 
+Configuring preset reasons for your server is simple. Please keep in mind that you must have the `Administrator` permission or be a Moderation Manager to configure preset reasons. 
 
 ### Managing Presets
 
-To set a new preset reason, do `/mod presetreason [variablename] (reason)`, to remove a preset reason do `/mod presetreason [variablename]` without a reason.
+To set a new preset reason, do `/mod presetreason [variablename] (reason)`, to remove a preset reason, do `/mod presetreason [variablename]` without a reason.
```

Comment:
```suggestion
To set a new preset reason, do `/mod presetreason [preset name] (reason)`, to remove a preset reason, do `/mod presetreason [preset name]` without a reason.
```
Unrelated to your PR, but saw this and thought it could use a clarity update.

Reply:
I was also wondering about the wording on the "-reasonhere" part. It made me a little confused on what you were actually supposed to put there (a number, reason, or actually putting "reasonhere")

Reply:
"Once added, all moderators in your server will now be able to use the preset reason you have configured by simply typing -reasonhere as their reason." (preset-reasons.md) it's a little confusing on what you're actually supposed to put there.

Reply:
Ohh, this is intended to be the variable name for the preset. So for example, the ban command is normally `/ban (user) (optional duration) (reason)`, you would replace `(reason)` with the preset reason variable, like `-boosting` @actuallypolar

Reply:
oh, that makes sense now. it was a bit confusing because i thought you still had to use /ban in front still